designs/administrator/olm/List-of-operators-in-all-projects/list-of-operators.md

```diff
@@ -0,0 +1,41 @@
+
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All projects view
+
+- When a user has selected the All Projects view from the context selector on the Installed Operators page, here they can view all of their Installed Operators. The goal of this design is to simplify how a user views the namespaces and associated operator(s) when they have selected to view the Installed Operators under All Projects.
+
+- When a user clicks the link in the table, “View 8…”, an anchor link takes the user to the Namespaces in the body of the Operator Details page. In this flow, we opt for a filterable experience in which the user can later filter by namespace or project under tabs such as Provided APIs or Events. (Scroll to view experience)
+
+- In an alternate path, if a user were to click into the Operator itself, it would take the user to the top of the Operator Details page rather than the anchor link in the body of the page. 
+
+![list-of-operators](img/list-of-operators.jpg)
+
+## Operator Details Page
+
+
+Here you can see the Namespaces in the body of the Operator Details page:
+
+![operator-details-page](img/operator-details-page.jpg)
+
+Here is the anchor link view:
+
+![namespaces-view](img/namespaces-view.jpg)
+
+Clicking into the kebab menu, the user can access a dropdown from kebab, and click “View Operator in this Namespace” to view this particular Namespace’s Operator.
+
+![menu-option](img/menu-option.jpg)
```

Comment:
Curious what's the next screen after clicking this dropdown menu? Would users be taken to the top of the Operator Details page in single-namespace view? Would be nice if we could clarify this a bit in here too.

Reply:
Agreed. I think if we are forwarding there I would probably rename this to "Use Operator in this Namespace".

Reply:
That's right Tony, the user would be taken to the top of the Operator Details page in single-namespace view - I can add that screen in